app/angular/service/conversorService.js

```diff
@@ -261,7 +261,8 @@ angular.module('myapp').factory('ConversorService', function(ConceptualService,
 					} else {
 							var element = tables.shift();
 							if(element != null) {
-								var promise = buildTable(element, modelGraph.getNeighbors(element));
+								const vizinhos = modelGraph.getNeighbors(element);
```

Comment:
Should we use **pt-br** as `const` name?
Maybe change it to something like `siblings` `neighbors`, `adjacents` or something similar?

Reply:
I was trying to understand why it wasn't working and I forgot to undo it. Thanks for watching!!